### ISSUE TITLE
WP2946 was removed from WikiPathways. Can this change propagate to OPS?

### ISSUE THREAD

**Issue by egonw** [openphacts/DATA#1]
What is the OPS approach for this?

**Comment by nicklynch** [comment]
Good question:  
has this happened before?
For small changes like this we would still need the the updated RDF to load
As for schedule, our aim would be next data release unless there was a strong case to change.
We could look to combine with another change

**Comment by AlexanderPico** [comment]
Indeed, this is a rare and urgent case. The pathway information was not intended to be made public yet. It contains not only unpublished results, but results that under legal (IP) consideration.  The original author (who intended to keep the pathway private until after publication) has made an impassioned plea to scrub as much as we can.

I apologize for the extra work, but if it is at all possible, it would be very much appreciated!

**Comment by nicklynch** [comment]
Can you suggest the best option to remove all the references to WP2946, we can then look at timing?

**Comment by AlexanderPico** [comment]
Interestingly, this particular page comes up in related Google searches. Near the bottom of the page is the description for WP2946: 
http://ops2.few.vu.nl/pathways.xml?pathway_organism=Homo%20sapiens&_metadata=all,views,formats,execution,bindings,site&_page=41

Perhaps that's a place to start? How is that page generated? Can it be updated or edited?

**Comment by egonw** [comment]
See Skype for details. I'm on it and let's coordinate and get/define answers there.
